<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configwindow</class>
 <widget class="QDialog" name="Configwindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration window</string>
  </property>
  <widget class="QStackedWidget" name="STK_WGT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>750</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="PG_INC">
    <widget class="QTableWidget" name="PG_INC_TBL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>750</width>
       <height>400</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Interval</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="BTN_PG_INC_SWT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOW EXPENSES LIST</string>
     </property>
    </widget>
    <widget class="QLabel" name="LBL_PG_INC">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INCOME LIST</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PG_EXP">
    <widget class="QTableWidget" name="PG_EXP_TBL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>750</width>
       <height>400</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Interval</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="BTN_PG_EXP_SWT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOW INCOME LIST</string>
     </property>
    </widget>
    <widget class="QLabel" name="LBL_PG_EXP">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>EXPENSES LIST</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="LINE_HOR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>750</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_EDIT">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ADD">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DAT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_DAT_TTL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save file from</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
